journalpump/senders/file.py

```diff
@@ -5,7 +5,7 @@ class FileSender(LogSender):
     def __init__(self, *, config, **kwargs):
         super().__init__(config=config, max_send_interval=config.get("max_send_interval", 0.3), **kwargs)
         self.mark_disconnected()
-        self.output = open(config["file_output"], "ab")
+        self.output = open(config["file_output"], "ab")  # pylint: disable=consider-using-with
```

Comment:
I think pylint warnings should be handled in a different commit. Also, do these pylint warnings mean there is technical debt in the implementation?

Reply:
Pylint 2.8, released at 2021-04-24, added a new refactoring message "consider-using-with" (http://pylint.pycqa.org/en/latest/whatsnew/2.8.html). The checker generates these messages
for roughly all calls that are often used with context managers. These cases seemed to me such
that it does not make sense to start changing them, so I added the directives here to get the code
to pass the linter.

Reply:
Yes, in `FileSender` the `request_stop()` should close the opened file as it is not done elsewhere. Good catch.

Reply:
This would be a good idea. I think that the log file handling could be put inside a context manager by overriding `run()` in `FileSender`.